apps/oauth2/src/components/OAuthItem.vue

```diff
@@ -87,6 +86,13 @@ export default {
 				return '****'
 			}
 		},
+		toggleAriaLabel() {
+			if (!this.renderSecret) {
+				return t('oauth2', 'Show client secret')
+			} else {
+				return t('oauth2', 'Hide client secret')
+			}
```

Comment:
```suggestion
			}
			return t('oauth2', 'Hide client secret')
```
Early return pattern instead of if-else :)